p5js/ImageClassification_TransferLearning/sketch.js

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) 2018 ml5
+//
+// This software is released under the MIT License.
+// https://opensource.org/licenses/MIT
+
+/* ===
+ML5 Example
+Image Classifier with Transfer Learning example with p5.js
+=== */
+
+let classifier;
+let video;
+let loss;
+let dogImages = 0;
+let catImages = 0;
+
+// Default options for training
+const options = {
+  learningRate: 0.0001,
+  hiddenUnits: 100,
+  epochs: 20,
+  numClasses: 2,
+  batchSize: 0.4,
+};
+
+function setup() {
+  noCanvas();
+  // Create a video element
+  video = createCapture(VIDEO, onVideoLoaded).parent('videoContainer');
```

Comment:
Chaining can sometimes be a little intimidating to beginners.

Reply:
yep, agree! will update then.